src/Builder.jl

```diff
@@ -38,38 +38,38 @@ abstract type DocumentPipeline <: Selectors.AbstractSelector end
 """
 Creates the correct directory layout within the `build` folder and parses markdown files.
 """
-abstract type SetupBuildDirectory <: DocumentPipeline end
+struct SetupBuildDirectory <: DocumentPipeline end
```

Comment:
Why are these changes necessary? I believe there are some packages that extend Documenter by subtyping some of these.